include/winrospublisher.h

```diff
@@ -127,7 +127,7 @@ namespace ros_msft_camera
             }
             catch (winrt::hresult_error const& ex)
             {
-                std::cout << ex.code() << ex.message().c_str();
+                std::wcout << ex.code() << ex.message().c_str();
```

Comment:
Can you make this a ROS_ERROR?

Reply:
(ROS_ERROR allows it to flow through the ROS logs which can be introspected using other tools)